doc/node.1

```diff
@@ -36,7 +36,10 @@ node \- Server-side JavaScript runtime
 .RI [ v8\ options ]
 .RI [ script.js \ |
 .B -e
-.RI \&" script \&"]
+.RI \&" script \&"
+.R |
+.B -
+.R ]
```

Comment:
Not had experience of doc writing using `troff` before, will apply whatever you conclude of course.